src/add.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] [OPTIONAL] You are ignoring the Linters check, please try to always fix errors instead.

Reply:
Yeah I'll try next time :pray: